task/ike-create/0.3/ike-create.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: ike-create
+  labels:
+    app.kubernetes.io/version: "0.3"
```

Comment:
should we add this as `0.1` version since there are no previous versions of this task?

Reply:
We had PRs for `0.1` and `0.2` but hence the linter has been updated in the meantime I thought I will close older ones as there's no point of using them anyway and only fix the latest one. See https://github.com/tektoncd/catalog/pull/726#issuecomment-844816786.

Setting it back to `0.1` will make our release process a bit clumsy, as we align the version of tasks (vX.Y) with our releases of the tool the tasks is using (vX.Y.Z), so it's always matching. Is it a big deal for you to start from `0.3`?